README.md

```diff
@@ -72,18 +83,21 @@ Notice that this system handle maps soss messages dyrectly to a JSON compatible
 
 ### More information
 
-- For more information, you can see the [demo steps](fiware/sample/demo.md),
+- For more information, you can see the [demo steps](fiware/doc/demo.md),
 and the related [video](https://drive.google.com/open?id=1w90DAPkovjwj7673d5RfOINlAAc7kWb1)
 - Also, you can have a look to the [internal design](fiware/doc/design.md)
-- For a fast configuration, you can use the [dockerfile](Dockerfile)
+- For a fast configuration, you can use the [dockerfile](Dockerfile). 
+**NOTICE**: Both when building and running the Dockerfile, you must specify the option `--network=host`, 
+because FIWARE SH tells Orion context broker its own IP in order to make subscriptions,
```

Comment:
The user maybe does not understands what is `SH`, maybe `SOSS FIWARE plugin` instead?

Reply:
I put "system handle" instead of SH